cache/clock_cache.cc

```diff
@@ -270,8 +270,11 @@ class ClockCacheShard final : public CacheShard {
   void SetStrictCapacityLimit(bool strict_capacity_limit) override;
   Status Insert(const Slice& key, uint32_t hash, void* value, size_t charge,
                 void (*deleter)(const Slice& key, void* value),
-                Cache::Handle** handle, Cache::Priority priority) override;
-  Cache::Handle* Lookup(const Slice& key, uint32_t hash) override;
+                Cache::Handle** handle, Cache::Priority priority,
+                const Slice& /*(*unpack)*/ (void* value),
```

Comment:
Return type of `unpack` should be `const Slice` instead of `const Slice&`.

Reply:
OK.

Reply:
Updated.